src/transformers/models/perceiver/modeling_perceiver.py

```diff
@@ -3166,9 +3170,13 @@ def forward(self, inputs: torch.Tensor, pos: Optional[torch.Tensor] = None, netw
         if self.prep_type != "patches":
             # move channels to last dimension, as the _build_network_inputs method below expects this
             if inputs.ndim == 4:
-                inputs = torch.moveaxis(inputs, 1, -1)
+                # equivalent to `torch.moveaxis(inputs, 1, -1)` but `torch.moveaxis`
+                # cannot be converted to ONNX
+                inputs = torch.permute(inputs, (0, 2, 3, 1))
             elif inputs.ndim == 5:
-                inputs = torch.moveaxis(inputs, 2, -1)
+                # equivalent to `torch.moveaxis(inputs, 2, -1)` but `torch.moveaxis`
+                # cannot be converted to ONNX
+                inputs = torch.permute(inputs, (0, 1, 3, 4, 2))
```

Comment:
I don't think we need the comments here. `moveaxis` is more esoteric than `permute` (which we use everywhere) :-)